src/theme/EditMetaRow/index.tsx

```diff
@@ -32,16 +32,15 @@ export default function EditMetaRow({ className, editUrl }: Props): JSX.Element
               <Button
                 className={styles.button}
                 onClick={() => {
-                  window.plausible?.('Docs Helpful');
+                  void fetch(
+                    'https://hooks.slack.com/triggers/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',
```

Comment:
nit: better define the webhook link in some place so that we can manage it in a easier way

Reply:
Updated